content/blog/set-up-apache-skywalking-for-k8s-monitoring-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,374 @@
+
+In this blog post, I will choose one APM tool,  *Apache SkyWalking*, and describe the detailed process how to set it up in HPE GreenLake for Private Cloud Enterprise for monitoring Kubernetes cluster.
+
+## Apache SkyWalking
+
+[Apache SkyWalking](https://skywalking.apache.org/) is an open source application performance monitor (APM) tool, especially designed for microservices, cloud native, and container-based architectures. 
+
+Apache SkyWalking is lightweight and scalable. It can be easily set up as a *self-managed* APM tool within an on-premises data center. This avoids leasing customer data to third party services and matches well with the strict security restriction in HPE GreenLake for Private Cloud Enterprise environment.
+
+## Prerequisites
+
+Before you start, make sure you meet the following requirements:
+
+* A Kubernetes cluster needs to be created in HPE GreenLake for Private Cloud Enterprise. You need to download the *kubectl* binary, together with the *HPE kubectl plugin* and the *kubeconfig* file of the created cluster, from the launched service console. The downloaded *kubectl* binary and its plugin need to be set up in your environment. To simplify the setup process, you should export the environment variable `KUBECONFIG` and point it to the downloaded kubeconfig file. With these setups in place, you can access the Kubernetes cluster in the HPE GreenLake for Private Cloud Enterprise.
+* The [Helm CLI](https://helm.sh/docs/intro/install/) needs to be installed in your environment. This Helm CLI will be used for installing and setting up the Apache SkyWalking.
+* The [istioctl](https://istio.io/latest/docs/setup/getting-started/#download) needs to be installed as well in your environment. The istioctl client will be used for installing and set up `Istio`.
+
+With your user access setup, you should have access to permissions that can create and update the following resources in the Kubernetes cluster:
+
+* Kubernetes Service Account(s)
+* Kubernetes Roles & RoleBindings
+
+## Set up Apache SkyWalking for Kubernetes monitoring
+
+### Prerequisites
+
+Before I start, make sure the following requirements a
+re met:
+
+* A Kubernetes cluster, being provisioned in HPE GreenLake for Private Cloud Enterprise;
+* The *kubectl* CLI tool, version 1.23 or later, together with the *kubeconfig* files for accessing the Kubernetes cluster; 
+* The *[Helm](https://helm.sh/docs/intro/install/)* CLI tool, version 3.8.1 or later. 
+
+Apache SkyWalking leverages Kubernetes [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) service for collecting metrics data from Kubernetes cluster. It then leverages *OpenTelemetry* collector to transfer the Kubernetes metrics to *OpenTelemetry* receiver for Kubernetes monitoring. 
+
+![](/img/otel-collector.png)
```

Comment:
If you have access to the original picture, would you mind adding the OTel "receiver" element in the picture.

Reply:
Good point. added it.